code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm

```diff
@@ -0,0 +1,47 @@
+//Los Mariachis - Grade 7 with poise crits, white version of Kurokumo.
+/obj/item/ego_weapon/city/mariachi
+	name = "maracas (single)"
+	desc = "A single maracas used by Los Mariachis."
+	special = "This weapon gains 1 poise for every attack. 1 poise gives you a 2% chance to crit at 3x damage, stacking linearly. Critical hits reduce poise to 0."
```

Comment:
Note: Not a requested change.
Optional that we make this like Limbus where crits at 20% "true" damage, not benefiting from weaknesses or resistances. In our case it'd likely be Brute.